fw-vsock/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
```

Comment:
Just wondering how this VSOCK implementation compares to the Firecracker VSOCK implementation used by Cloud Hypervisor?

Reply:
Do you mean https://github.com/firecracker-microvm/firecracker/blob/main/src/devices/src/virtio/vsock/device.rs ?

We evaluated in the beginning. We want to reuse but failed.

The Firecracker VSOCK has STD dependency. But our env is bare-metal, without STD.

Reply:
I think probably the virtio and vsock to too early to put here.
I will withdraw them.

Reply:
@jyao1 - yes, that's the crate used by CH here I believe: https://github.com/cloud-hypervisor/cloud-hypervisor/blob/main/virtio-devices/src/vsock/device.rs. Thanks for the update.